canonical-web-and-design/snapcraft.io
# Brandstore settings form

## Done
- Added the settings form for the brand store

## QA
- Go to https://snapcraft-io-3674.demos.haus/admin/ahnuP3quahti9vis8aiw/settings
- Check that the save button is enabled when fields have changed
- Check that saving the form works as expected
- You can safely ignore reviewing the settings tests as I'll continue this in a separate PR as this one is already fairly big

## Issue
Fixes #3661
